docs/posts/0000-00-00-template.md

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
This file serves as a template for the posts we create, it has a frontmatter section that need to be filled out besides the post itself.

This metadata must be correctly filled out, as these are used by the index to render the cards that allow access to each post.

- date: ISO format, time is not necessary (unless we want to display them in the publications)
- title: Post's title
- description: Introduction to the post, usually the first paragraph of it
- coverUrl: Cover image of the post
- coverAlt: Description of the post's cover image